aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1501,6 +1501,10 @@ Tensor _lu_solve_helper_cuda(const Tensor& self, const Tensor& LU_data, const Te
   auto self_working_copy = cloneBatchedColumnMajor(self);
   auto LU_data_working_copy = cloneBatchedColumnMajor(LU_data);
   auto LU_pivots_working_copy = LU_pivots.is_contiguous() ? LU_pivots : LU_pivots.contiguous();
+
+  if (self.numel() == 0 || LU_data.numel() == 0) {
```

Comment:
Is there a test case for the zero-dim case?

Reply:
I'll add that as well. Thank you.